pkg/controller/command/vdri/command.go

```diff
@@ -30,41 +34,72 @@ const (
 
 	// CreatePublicDIDError is for failures while creating public DIDs
 	CreatePublicDIDError
+
+	// SaveDIDErrorCode for save did error
+	SaveDIDErrorCode
+
+	// GetDIDErrorCode for get did error
+	GetDIDErrorCode
+
+	// GetDIDByNameErrorCode for get did  by name error
+	GetDIDByNameErrorCode
 )
 
 const (
 	// command name
 	commandName = "vdri"
 
+	// command methods
+	createPublicDIDCommandMethod = "CreatePublicDID"
+	saveDIDCommandMethod         = "SaveDID"
+	getDIDsCommandMethod         = "GetDIDs"
+	getDIDCommandMethod          = "GetDID"
+	getDIDByNameCommandMethod    = "GetDIDByName"
+
 	// error messages
 	errDIDMethodMandatory = "invalid method name"
+	errEmptyDIDName       = "did name is mandatory"
```

Comment:
name is mandatory.
`did name` is confusing

Reply:
done